cea/functions.py

```diff
@@ -695,12 +696,24 @@ def CalcThermalLoads(Name, prop_occupancy, prop_architecture, prop_thermal, prop
         Vol_ls = Lsww_dis*(gv.D/1000)**(2/4)*math.pi
         Qww_ls_r  = np.vectorize(calc_Qww_ls_r)(Ta, Qww, Lsww_dis, Lcww_dis, Y[1], Qww_0, Vol_ls, gv.Flowtap, Tww_sup_0, gv.Cpw , gv.Pwater)
         Qww_ls_nr  = np.vectorize(calc_Qww_ls_nr)(Ta, Qww, Lvww_dis, Lvww_c, Y[0], Qww_0, Vol_ls, gv.Flowtap, Tww_sup_0, gv.Cpw , gv.Pwater, gv.Bf, T_ext)
-    
-        
-        # Calc requirements of generation systems for hot water - assume losses of 10% due to local storage
-        Qwwf = (Qww+Qww_ls_r+Qww_ls_nr)/0.9
-        Qwwf_0 = Qwwf.max()    
-        
+
+        # fully mixed storage tank sensible heat loss calculation
+        Qww_ls_st = np.zeros(8760)
+        Tww_st = np.zeros(8760)
+        Qd = np.zeros(8760)
+        Qwwf = np.zeros(8760)
+        Vww_0 = Vww.max()  # peak dhw demand in m3/hour, also used for dhw tank sizing.
+        Tww_st_0 = gv.Tww_setpoint  #initial tank temperature in C
```

Comment:
@shanshanhsieh for the new issue about control systems #159 , make sure the seed of Twwst_0 is equal to Tsww0_C